src/github_proxy.py

```diff
@@ -16,7 +16,7 @@
 
 HIDDEN_COMMENT = """
 <!--
-CREATED BY GITHUB-CODBUILD-LOGS
+CREATED BY GITHUB-CODEBUILD-LOGS
```

Comment:
Nice catch, missed that

Reply:
Yeah glad I noticed it before publishing a new version. Then it would have been set in stone for backwards-compatibility. 😊